### PR TITLE
Add query_str to path

### PR DESCRIPTION
**Pull request by LucaNicosia**
## Description

The `query_str` parameter was missing in path

## Type of change

- Bug fix (non-breaking change which fixes an issue)
## Double Check

- [x] I have read the comments and followed the [CONTRIBUTING.md](https://github.com/netenglabs/suzieq/blob/master/CONTRIBUTING.md).
- [x] I have explained my PR according to the information in the comments or in a linked issue.
- [x] My PR source branch is created from the `develop` branch.
- [x] My PR targets the `develop` branch.
- [x] All my commits have `--signoff` applied

### CODE REVIEW

File: suzieq/cli/sqcmds/PathCmd.py [hunk 1]
```diff
@@ -25,6 +25,7 @@ def __init__(
         src: str = "",
         dest: str = "",
         vrf: str = "",
+        query_str: str = ' ',
```
Comment (claudiolor) [inline suzieq/cli/sqcmds/PathCmd.py:28]: Are you passing a string with an empty char on purpose or it is a mistake?

Comment (ddutt) [reply suzieq/cli/sqcmds/PathCmd.py:28]: All query_str have an blank char string. I don't recall why its so.

Comment (ddutt) [reply suzieq/cli/sqcmds/PathCmd.py:28]: Oh I remember why. One of the pandas' versions had an issue with the query-str if we didn't end with a blank. I don't know if this is still the case with the 1.3.4 version we're using.

File: suzieq/engines/pandas/path.py [hunk 2]
```diff
@@ -1120,7 +1121,8 @@ def get(self, **kwargs) -> pd.DataFrame:
             # This occurs when a path traversal terminates due to an error such
             # as loop detected
             final_paths = paths
-        return self._path_cons_result(final_paths)
+        return self._handle_user_query_str(
+            self._path_cons_result(final_paths), query_str)
```
Comment (ddutt) [inline suzieq/engines/pandas/path.py:1125]: You need to add reset_index for this to work correctly.

**Review by ddutt — CHANGES_REQUESTED** [review]

**Review by ddutt — APPROVED** [review]

**Pull request merged**